spring-cloud-stream/src/main/java/org/springframework/cloud/stream/aggregate/AggregateApplicationUtils.java

```diff
@@ -33,63 +34,52 @@
  * @author Venil Noronha
  */
 abstract class AggregateApplicationUtils {
-
-	private static final String SPRING_CLOUD_STREAM_INTERNAL_PREFIX = "spring.cloud.stream.internal";
-
-	private static final String CHANNEL_NAMESPACE_PROPERTY_NAME =
-			SPRING_CLOUD_STREAM_INTERNAL_PREFIX + ".channelNamespace";
-
-	private static final String SELF_CONTAINED_APP_PROPERTY_NAME =
-			SPRING_CLOUD_STREAM_INTERNAL_PREFIX + ".selfContained";
-
 	public static final String INPUT_CHANNEL_NAME = "input";
 
 	public static final String OUTPUT_CHANNEL_NAME = "output";
 
-	static ConfigurableApplicationContext createParentContext(Object[] sources, String[] args, final
-			boolean selfContained, boolean webEnvironment,
+	static ConfigurableApplicationContext createParentContext(Object[] sources,
+			String[] args, final boolean selfContained, boolean webEnvironment,
 			boolean headless) {
 		SpringApplicationBuilder aggregatorParentConfiguration = new SpringApplicationBuilder();
-		aggregatorParentConfiguration
-				.sources(sources)
-				.web(webEnvironment)
+		aggregatorParentConfiguration.sources(sources).web(webEnvironment)
 				.headless(headless)
 				.properties("spring.jmx.default-domain="
-									+ AggregateApplicationBuilder.ParentConfiguration.class.getName(),
-						SELF_CONTAINED_APP_PROPERTY_NAME + "=" + selfContained);
+						+ AggregateApplicationBuilder.ParentConfiguration.class.getName(),
+						InternalPropertyNames.SELF_CONTAINED_APP_PROPERTY_NAME + "="
+								+ selfContained);
 		return aggregatorParentConfiguration.run(args);
 	}
 
 	static String getDefaultNamespace(String appClassName, int index) {
 		return appClassName + "_" + index;
 	}
 
-
 	protected static SpringApplicationBuilder embedApp(
 			ConfigurableApplicationContext parentContext, String namespace,
 			Class<?> app) {
-		return new SpringApplicationBuilder(app)
-				.web(false)
-				.main(app)
-				.bannerMode(Mode.OFF)
+		return new SpringApplicationBuilder(app).web(false).main(app).bannerMode(Mode.OFF)
 				.properties("spring.jmx.default-domain=" + namespace)
-				.properties(CHANNEL_NAMESPACE_PROPERTY_NAME + "=" + namespace)
-				.registerShutdownHook(false)
-				.parent(parentContext);
+				.properties(
+						InternalPropertyNames.NAMESPACE_PROPERTY_NAME + "=" + namespace)
+				.registerShutdownHook(false).parent(parentContext);
 	}
 
-	static void prepareSharedChannelRegistry(SharedChannelRegistry sharedChannelRegistry,
+	static void prepareSharedBindingTargetRegistry(
+			SharedBindingTargetRegistry sharedChannelRegistry,
```

Comment:
the method param name should probably change even if this impl is still currently opinionated toward channels